packages/admin-stories/src/admin/table/TableWithFilterBar.tsx

```diff
@@ -0,0 +1,160 @@
+import { Field, FilterBar, FinalFormInput, Table, TableFilterFinalForm, TableQuery, useTableQuery, useTableQueryFilter } from "@comet/admin";
+import { FinalFormReactSelectStaticOptions } from "@comet/admin-react-select";
+import { Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as qs from "qs";
+import * as React from "react";
+
+import { apolloStoryDecorator } from "../../apollo-story.decorator";
+
+const gqlRest = gql;
+
+const query = gqlRest`
+query users(
+    $pathFunction: any
+    $username: String
+    $name: String
+    $email: String
+    $website: String
+) {
+    users(
+        email: $email
+        username: $username
+        name: $name
+        website: $website
+    ) @rest(type: "User", pathBuilder: $pathFunction) {
+        id
+        name
+        username
+        email
+        phone
+        website
+    }
+}
+`;
+function pathFunction({ args }: { args: { [key: string]: any } }) {
```

Comment:
Shouldn't we simplify the filterbar stories by not using a remote source?

Reply:
you have a suggestion on which data we should use in our stories in general ? or is my filterbar story the only one who uses this remote source ?

Reply:
I'd suggest creating some helpers to simplify data loading for the stories (in another PR)

Reply:
we will discuss that today in our meeting :)